package.json

```diff
@@ -6,12 +6,28 @@
   "description": "Preview SDK for both the field tagging connection + live content updates",
   "source": "./src/index.tsx",
   "main": "./dist/index.cjs",
-  "module": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "module": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
   "type": "module",
   "files": [
-    "dist"
+    "dist",
+    "react",
+    "style.css"
   ],
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
```

Comment:
we could add the "types": "./dist/index.d.ts", as it should now  work since ts@4.9

Reply:
Does that mean that all of our customers need to be on that version of ts?

Reply:
mh you're right that would be the requirement